partials/forms/radio-group.html

```diff
@@ -1,4 +1,4 @@
-<div id="{{key}}-group" class="form-group{{#className}} {{className}} {{/className}}{{#error}} error{{/error}}">
+<div id="{{key}}-group" class="form-group{{#className}} {{className}} {{/className}}{{#error}} error{{/error}}" {{#attributes}} {{attribute}}="{{value}}" {{/attributes}}>
```

Comment:
I think the spaces before the opening and closing `attributes` tags can be removed.